### PR TITLE
Ci project 4 delivery

### PR DESCRIPTION
**Pull request by chrisignas**

### CODE REVIEW

**Review by fpatelm — CHANGES_REQUESTED** [review]
I don't see this class CountSymptoms on Git.

File: Project02Eclipse/src/com/hemebiotech/analytics/SymptomsWriter.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package com.hemebiotech.analytics;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
```
[the remark this answers is not in the document]
Comment (chrisignas) [reply Project02Eclipse/src/com/hemebiotech/analytics/SymptomsWriter.java:6]: CountSymptoms should be added now.

File: Project02Eclipse/src/com/hemebiotech/analytics/CountSymptoms.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
+package com.hemebiotech.analytics;
+
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+
+/**
+ * <h1>Count Symptoms Class</h1>
+ * The CountSymptoms program implements an application that
+ * simply counts occurrences of symptoms within a text file
```
Comment (fpatelm) [inline Project02Eclipse/src/com/hemebiotech/analytics/CountSymptoms.java:10]: replace "within a text file" by "within a list of strings (symptoms)"

Comment (chrisignas) [reply Project02Eclipse/src/com/hemebiotech/analytics/CountSymptoms.java]: Thank you, corrected.

**Pull request merged**